src/components/config/validate.ts

```diff
@@ -435,3 +445,22 @@ function validateTCPConfig(tcpConfig?: Socket) {
 
   return validationError?.error?.message
 }
+
+function validateRedisConfig(redisConfig?: Redis[]) {
+  if (!redisConfig) {
+    return ''
+  }
+
+  const schema = Joi.object({
+    host: Joi.string().required(),
```

Comment:
Wonder if we can validate `host` to be hostname or ip address like this 

```
    host: Joi.valid([Joi.string().hostname(), Joi.string().ip()]).required(),

```

Reply:
TIL 😁 
`host: Joi.alternatives().try(Joi.string().hostname(), Joi.string().ip()).required()`